### PR TITLE
WIP: Updating config.json

### PR DESCRIPTION
**Pull request by IanWhitney**
Working on #179

The new exercises collection is in place and I'm populating the topics
values.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -67,126 +67,189 @@
       "slug": "hamming",
       "difficulty": ,
       "topics": [
+        "Exception handling",
+        "Result",
+        "Higher-orderfunctions",
```
Comment (petertseng) [inline config.json]: add space

File: config.json [hunk 2]
```diff
@@ -67,126 +67,189 @@
...
       "topics": [
+        "Enumerations",
+        "Immutability"
       ]
     },
     {
       "slug": "bracket-push",
       "difficulty": ,
       "topics": [
+        "From trait"
```
Comment (petertseng) [inline config.json]: comma

File: config.json [hunk 3]
```diff
@@ -3,6 +3,322 @@
...
+      "slug": "roman-numerals",
+      "difficulty": ,
+      "topics": [
+        "From trait",
+        "Display trait",
+        "Maps"
+      ]
+    },
+    {
+      "slug": "hexadecimal",
```
Comment (petertseng) [inline config.json]: (just happened) we kicked out hexadecimal for all-your-base, replace here

**Comment by petertseng** [comment]
There has not been much activity here for a while.

What would you say if I were to take over it?

My plan is to just use problems.md exactly for the topics, and a difficulty of 1 for everything. We can refine both items later.

**Comment by petertseng** [comment]
Just as #179 states, we should take it in steps. Therefore, my proposal.

**Comment by IanWhitney** [comment]
The problem I ran in to is that I find the difficulty rating impossible. What does 'difficult' mean? Is it hard because it introduces Lifetimes? Because it's algorithmically hard? And guessing at the difficulty of exercises I haven't done was even harder.

Starting everything with a difficulty of 1 is fine.

**Comment by IanWhitney** [comment]
Replaced by #229. closing.

**Pull request closed without merging**